### PR TITLE
Update/dialog

### PR DESCRIPTION
**Pull request by zsotroav**
Update the dialog command to include new characters, backgrounds, and ribbons. Also resized all images to avoid having to resize them on the fly upon every rendering, gaining performance and shrinking down on unnecessary file sizes.

### CODE REVIEW

File: utils/utils.py [hunk 1]
```diff
@@ -94,28 +93,15 @@ def get_first_name(full_name: str) -> str:
     return full_name.split(' ')[0]
 
 
-def render_dialog(text: str, character: str, background: str = 'camp') -> BytesIO:
+def render_dialog(text: str, charactername: str, background: str = 'camp') -> BytesIO:
     background = Image.open('./assets/images/dialog/backgrounds/' + background + '.png')
-    ribbon = Image.open('./assets/images/dialog/ribbons/' + character + '.png')
-    character = Image.open('./assets/images/dialog/characters/' + character + '.png')
+    ribbon = Image.open('./assets/images/dialog/ribbons/' + charactername + '.png')
```
Comment (duncte123) [inline utils/utils.py]: You should use the character aliases we have for the quote command, they can be moved to this file and imported where needed

**Review by duncte123 — CHANGES_REQUESTED** [review]
File name suggestions:
bar => stage
lobby => hotel_lobby
meeting_room => clermont_office
newsite => camp2
quarters => cabin2
shed => shed_out

File: utils/utils.py [hunk 2]
```diff
@@ -16,6 +16,18 @@
 font = ImageFont.truetype('./assets/fonts/YuuPy_combined.ttf', 56)
 flag = Image.open('./assets/images/dialog/flag_overlay.png')
 textbox = Image.open('./assets/images/dialog/text_box.png')
+char_ribbon_alias = {
```
Comment (duncte123) [inline utils/utils.py]: I suggest that you send over the converted names so that you don't have to parse as much here

Comment (duncte123) [reply utils/utils.py]: Other than that it looks good

Comment (duncte123) [reply utils/utils.py]: I think a better option with be to strip `young_` from the string

Comment (zsotroav) [reply utils/utils.py]: jin and yoshi are already processed, that should be removable, but I don't know what would be the best thing to do about the `young_` part.

**Review by duncte123 — APPROVED** [review]
LGTM

**Pull request merged**